extensions/amp-springboard-player/0.1/amp-springboard-player.js

```diff
@@ -182,6 +182,7 @@ class AmpSpringboardPlayer extends AMP.BaseElement {
     } else {
       placeholder.setAttribute('alt', 'Loading video');
     }
+    this.applyFillContent(placeholder);
```

Comment:
Remove the `setAttribute('layout', 'fill')`?

Reply:
Whoops, missed this one.